index.rst

```diff
@@ -48,14 +48,261 @@
+#. Simplicity. There may be many functional forms that can meet our accuracy requirements. Here the criteria is to make sure the error due to finite number of coefficients is much less than the non-repeating error of the corresponding degree of freedom (DOF). Document-36395 :cite:`Document-36395` shows an analysis that demostrates that the 5th order standard polynomials are accurate enough in all cases. And it is the simplest form among the above.
+#. Various requirement documents (for example, LTS-88 :cite:`LTS-88` and LTS-206 :cite:`LTS-206`) specify that 5th order standard polynomials be used. We do not want to go through the change request process unless it is necessary.
+
+##########################
+Actuator weight components
+##########################
+
+For M1M3 and M2, the actuators have to supply the forces that can support their own weights first. Additional forces are then used to support and shape the glass. The LUTs therefore need to have actuator weight components. These forces can be calculated analytically, although they might not be as straightforward as one might first expect.
+
+The calculations for M1M3 actuators (both single-axis and dual-axis actuators) are given in Document-32192 :cite:`Document-32192`. Measurements were also performed for standalone M1M3 actuators on the test bench in Tucson :cite:`Document-34907`. The results were consistent with the analytical calculations within measurement errors. We therefore use the results from the calculations in the final actuator weight component of the M1M3 LUT.
+
+The M2 LUT actuator weight component was supplied by the M2 vendor Harris. The Rubin team plans to recalculate these forces to crosscheck the Harris results.
+
+The M2 and the camera hexapods do not have actuator weight components in their LUTs because the output of the hexapod LUTs are positions instead of forces.
+
+#################
+Static components
+#################
+
+The static component of the LUT doesn't vary with external conditions. For the mirrors, these are the forces that are needed to bend out the low spatiall frequency factory figuring error. These were supplied by the vendors during factory acceptance testings. We will not change these components during commissioning and operations, unless somehow it can be proven that the figuring errors are different from what were determined at the factories.
+
+As for the hexapods, the :math:`C_0` defined in Sec. :ref:`sec-p5` is the static component. For now, all six coefficients for the 5th order standard polynomial for each hexapod have been determined using results from FEA analyses. Once we have the hexapods mounted on the telescope mount assembly (TMA), we will use Laser Trackers (LTs) to calibrate the LUTs for both hexapods. It is expected that the calibrated values of :math:`C_0` will be quite different from the FEA values, because they depend on the actual installations of the hexapods on the TMA.
+
```

Comment:
At what temperature do we measure C0 for the hexapods?

Reply:
Very good point. This needs to be defined at a reference temperature (Tref). I believe we need to use the same Tref for all subsystems. Tref=21C seems to be a reasonable choice (This is what Harris has been using for M2). I will edit accordingly.